docs/source/contributing.md

```diff
@@ -2,154 +2,88 @@
-```
+To develop a scylla-operator, your environment must have the following:
+
+1. [Go](https://golang.org/dl)
+   * Check `go.mod` for minimal version requirement.
+2. Git client
+3. GitHub account
 
 ## Initial Setup
 
 ### Create a Fork
 
-From your browser navigate to [http://github.com/scylladb/scylla-operator](http://github.com/scylladb/scylla-operator) and click the "Fork" button.
-
-### Clone Your Fork
-
-Open a console window and do the following:
-
-```bash
-# Create the scylla operator repo path
-mkdir -p $GOPATH/src/github.com/scylladb
-
-# Navigate to the local repo path and clone your fork
-cd $GOPATH/src/github.com/scylladb
-
-# Clone your fork, where <user> is your GitHub account name
-git clone https://github.com/<user>/scylla-operator.git
-```
-
-### Add Upstream Remote
-
-First you will need to add the upstream remote to your local git:
-```bash
-# Add 'upstream' to the list of remotes
-git remote add upstream https://github.com/scylladb/scylla-operator.git
-
-# Verify the remote was added
-git remote -v
-```
-Now you should have at least `origin` and `upstream` remotes. You can also add other remotes to collaborate with other contributors.
+Follow GitHub documentation about creating a fork: https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/fork-a-repo
 
 ## Development
 
 To add a feature or to make a bug fix, you will need to create a branch in your fork and then submit a pull request (PR) from the branch.
+A subset of useful commands:
+* `make build` - build project binaries
+* `make test-unit` - run unit tests
+* `make update` - run all code generators
+* `make verify` - verify if all autogenerated changes are generated.
 
-### Building the project
+Check `make help` for more.
 
-You can build the project using the Makefile commands:
-* Open the Makefile and change the `IMG` environment variable to a repository you have access to.
-* Run `make docker-push` and wait for the image to be built and uploaded in your repo.
+## Coding convention
 
-### Create a Branch
-
-From a console, create a new branch based on your fork and start working on it:
-
-```bash
-# Ensure all your remotes are up to date with the latest
-git fetch --all
-
-# Create a new branch that is based off upstream master.  Give it a simple, but descriptive name.
-# Generally it will be two to three words separated by dashes and without numbers.
-git checkout -b feature-name upstream/master
-```
+This outlines guidelines, style suggestions, and tips for writing code throughout the Operator project.
+The new code should follow the rules outlined in these:
+* [Effective Go](https://go.dev/doc/effective_go)
+* [Go Code Review Comments](https://go.dev/wiki/CodeReviewComments)
+* [API changes](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api_changes.md)
+* [API conventions](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md)
```

Comment:
It could also include logging convention https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/logging.md but I don't feel strong since it's pretty clear from existing usage